src/__test__/e2e/signing/signing.test.ts

```diff
@@ -28,8 +28,8 @@ describe('Test General Signing', () => {
     }
   });
 
-  runDeterminismTests({ client })
-  runUnformattedTests({ client })
-  runSolanaTests({ client });
+  // runDeterminismTests({ client })
```

Comment:
some commented out tests here

Reply:
Yeah if you uncomment those out it will blow up because the wallet jobs handlers haven't been updated or something. Will look into that next.

Reply:
Fixed in [55c49a5](https://github.com/GridPlus/gridplus-sdk/pull/450/commits/55c49a5a80d7de6321c95cca09ca39383969b022)